.github/workflows/integration-tests.yml

```diff
@@ -226,10 +226,66 @@ jobs:
           load: true
           build-args: |
             PROFILE=ci
+      - name: Install dependencies
+        run: sudo snap install yq
+      - name: Run test snapshot operations on different storage filesystems
+        working-directory: ./tests/snapshots
+        shell: bash
+        run: ./snapshots-recovery.sh local
+
+  test-snapshot-operations-s3-minio:
+    runs-on: ubuntu-latest
+
+    # Setup minio server
+    services:
+      minio:
+        image: wktk/minio-server
+        ports:
+          - 9000:9000
+        env:
+          MINIO_ACCESS_KEY: "minioadmin"
+          MINIO_SECRET_KEY: "minioadmin"
+
```

Comment:
almost same as `test-snapshot-operations-local:` but added minio server so that we can use `minio` from the shell file.
I refered https://github.com/mozilla/sccache/pull/1513 to implement this.